tests/std/tests/P1135R6_atomic_wait/test.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _ENABLE_ATOMIC_ALIGNMENT_FIX
+
+#include <atomic>
+#include <cassert>
+#include <chrono>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+template <class UnderlyingType>
+void test_atomic_wait_func(
+    UnderlyingType old_value, UnderlyingType new_value, std::chrono::steady_clock::duration waiting_duration) {
+
+    std::string seq;

```

Comment:
NB: Can be done without a mutex like this:

```
char buf[20];
atomic<char*> base = &buf[0]; // initially
add_seq = [&base](char c) { base.fetch_add(1)[0] = c; };
```

You don't need to do that if you don't want to.

Reply:
Why don't, I like it